test/test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+# test script to control starting and running everything in right order.
+
+docker compose up --build bastion -d
+docker compose exec bastion /setup.sh
+docker compose run --build test /client_test.sh
+
+retVal=$?
+
+if [ $retVal -ne 0 ]; then
+  echo "* Google Authenticator/SSH Test Failed"
+  exit $retVal
+else
+  echo "* Google Authenticator/SSH Test Succeeded"
+fi 
+
+
+docker compose exec bastion ls /var/log/sudo-io/00/00/01/ 
+
+retVal=$?
+
+if [ $retVal -ne 0 ]; then
+  echo "* sudosh Audit Failed - no logs created!"
+  exit $retVal
+else
+  echo "* sudosh Audit Test Succeeded"
+fi
+
+
+docker compose exec bastion curl https://hooks.slack.com
```

Comment:
What was the test case for this? resolv.conf broken or something?

Reply:
https://github.com/cloudposse/bastion/issues/67 - this is to catch the Root CA certs being out of date issue and validate/verify that the slack hook functionality is not broken.